capstone/src/components/Visitor/Login/Login.jsx

```diff
@@ -5,11 +5,12 @@ import './Login.css';
 
 export function logIn(formValues) {
   const baseURL = process.env.REACT_APP_BASE_URL;
-  const values = {
-    username: formValues.username,
-    password: formValues.password,
-  };
-  return axios.post(`${baseURL}/visitor/logIn`, values);
+  return axios.post(`${baseURL}/visitor/logIn`, {}, {
+    headers: {
+      username: formValues.username,
+      password: formValues.password,
+    },
+  });
```

Comment:
Just wondering, don't have context for discussions that have happened, any reason username and password are being sent as headers now instead of body?

Reply:
Oh, sure! 
I got this feedback where I learned that passing that information through a req.body can allow malicious users to impersonate someone else, so that it would be a better practice to pass it through the headers or cookies, and it was an easier transition to do it through headers 😅 https://github.com/carlosvegap/capstone-kickoff/pull/45#discussion_r932559554

Reply:
Ah, to clarify I was just wondering why it was necessary to submit `username` directly in a non-login-related request (`submitRate` in this case).

Reply:
Ah, yeah, to clarify on that as well, a malicious user could impersonate anything about the request _(from query params, to headers, to request body, etc.)_. 

I wouldn't spend time on building proper auth, you'd need to get into session tokens or JWTs unless Parse has something builtin, but just fyi both headers/body would be insecure.

Reply:
Ohhhh! That is really interesting. 
I will try to add a proper auth for that! Thanks for the clarification